web/src/components/tasks/LicenseCurrentStatusComponent.tsx

```diff
@@ -0,0 +1,96 @@
+import { ChecklistTag } from "@/components/ChecklistTag";
+import { HorizontalLine } from "@/components/HorizontalLine";
+import { Icon } from "@/components/njwds/Icon";
+import { LicenseStatusHeader } from "@/components/tasks/LicenseStatusHeader";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { parseDateWithFormat } from "@businessnjgovnavigator/shared/dateHelpers";
+import { defaultDateFormat } from "@businessnjgovnavigator/shared/defaultConstants";
+import {
+  LicenseData,
+  LicenseName,
+  licenseSearchDateFormat,
+  LicenseStatusItem,
+} from "@businessnjgovnavigator/shared/license";
+import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
+import { ReactElement } from "react";
+
+interface Props {
+  licenseData?: LicenseData;
+  licenseName: LicenseName;
+}
+
+export const LicenseCurrentStatusComponent = (props: Props): ReactElement => {
+  const { Config } = useConfig();
+
+  if (!props.licenseData) return <></>;
+  const licenseDetails = props.licenseData?.licenses?.[props.licenseName];
+
+  const getOneLineAddress = (): string => {
+    const nameAndAddress = licenseDetails?.nameAndAddress;
+    if (!nameAndAddress) {
+      return "";
+    }
+    const secondLineAddress = nameAndAddress.addressLine2 ? ` ${nameAndAddress.addressLine2}` : "";
+    return `${nameAndAddress.addressLine1}${secondLineAddress}, ${nameAndAddress.zipCode} NJ`.toUpperCase();
+  };
+
+  const receiptItem = (item: LicenseStatusItem, index: number): ReactElement => {
+    return (
+      <div key={index} data-testid={`item-${item.status}`}>
+        <div
+          className={`flex flex-column fac tablet-flex-row width-full ${
+            index === 0 ? "" : "border-top-1px padding-top-2 tablet:padding-top-05"
+          }  border-base-lightest padding-bottom-2 tablet:padding-bottom-05`}
+        >
+          <ChecklistTag status={item.status} />
+          <span className="tablet:margin-left-2 text-left width-full">{item.title}</span>
+        </div>
+      </div>
+    );
+  };
+
+  return (
+    <div data-testid="license-current-status-component">
+      <div className="h3-styling">{Config.currentLicenseStatusDefaults.licenseStatusHeaderText}</div>
```

Comment:
pebble - should this actually be an H3? or at least a heading tag of some kind?

Reply:
added h2 heading with h3 variant